core-bundle/tests/Image/PictureFactoryTest.php

```diff
@@ -250,6 +253,7 @@ function (ResizeOptions $options): bool {
 
         $imageFactory = $this->createMock(ImageFactoryInterface::class);
         $imageFactory
+            ->expects($this->once())
```

Comment:
`$imageFactory` Can be removed completely here.
It would only be needed if `$pictureFactory->create()` is called with a path instead of an image object which is not the case here.

Reply:
👍  removed in ea2ab19dfe8fa433393adb03143c4def46a69ce4